src/patternfly/components/Table/table.scss

```diff
@@ -638,26 +643,65 @@
       --pf-c-table__compound-expansion-toggle__button--before--BorderLeftWidth: 0;
     }
   }
+
+  &:active,
+  &:focus-within {
+    outline-offset: var(--pf-c-table__button--OutlineOffset);
+
+    // stylelint-disable media-feature-name-no-vendor-prefix
+    @media (-webkit-min-device-pixel-ratio: 0) {
+      outline-style: auto;
+      outline-color: -webkit-focus-ring-color;
+    }
+    // stylelint-enable
+  }
+}
+
+.pf-c-table__column-help {
+  display: flex;
+  align-items: flex-end;
+}
+
+.pf-c-table__column-help-action {
+  margin-left: var(--pf-c-table__column-help--MarginLeft);
+  transform: translateY(var(--pf-c-table__column-help--TranslateY));
+
+  .pf-c-button {
+    --pf-c-button--PaddingRight: var(--pf-c-table__column-help--c-button--PaddingRight);
+    --pf-c-button--PaddingLeft: var(--pf-c-table__column-help--c-button--PaddingLeft);
+
+    margin-top: var(--pf-c-table__column-help--c-button--MarginTop);
+    margin-bottom: var(--pf-c-table__column-help--c-button--MarginBottom);
+    font-size: inherit;
+    line-height: 1;
+  }
 }
 
 // Table sort
 // ==================================================================
 .pf-c-table__sort {
-  .pf-c-table__button:hover {
-    --pf-c-table__sort-indicator--Color: var(--pf-c-table__sort__button--hover__sort-indicator--Color);
-  }
+  --pf-c-table__button--PaddingTop: var(--pf-c-table__sort__button--PaddingTop);
+  --pf-c-table__button--PaddingRight: var(--pf-c-table__sort__button--PaddingRight);
+  --pf-c-table__button--PaddingBottom: var(--pf-c-table__sort__button--PaddingBottom);
+  --pf-c-table__button--PaddingLeft: var(--pf-c-table__sort__button--PaddingLeft);
 
-  .pf-c-table__button:focus {
-    --pf-c-table__sort-indicator--Color: var(--pf-c-table__sort__button--focus__sort-indicator--Color);
-  }
+  .pf-c-table__button {
+    display: inline-flex;
```

Comment:
do you need this

Reply:
It was `display: flex`, so I made it `inline-flex` so that it wouldn't take up the width of the cell... but I refactored that bit so it can stay `display: flex;`. I'm not sure if that's needed for one of the modifiers, but I don't see any harm in leaving it even if it isn't necessary. It will remove any gap between its children (the button text and indicator icon), and allow for better alignment (using flex properties) if that's ever needed.